frontend/src/components/widget/Attributes/AttributesDropdown.js

```diff
@@ -125,8 +130,17 @@ class AttributesDropdown extends Component {
    * @todo Write the documentation
    */
   render() {
+    const { isVisible, dropdownCoords } = this.state;
+
     return (
-      <div className="attributes-dropdown panel-shadowed panel-primary panel-bordered panel-spaced">
+      <div
+        ref={this.selector}
+        className={classnames(
+          'attributes-dropdown panel-shadowed panel-primary panel-bordered panel-spaced',
+          { 'hidden ': !isVisible },
+          { 'attributes-dropup': dropdownCoords && dropdownCoords.y > 680 }
```

Comment:
pls extract 680 as `const` at top of this file.
In this way it would be easier to track/tune hardcoded constants.